x/pairing/keeper/pairing_test.go

```diff
@@ -62,6 +63,17 @@ func TestPairingUniqueness(t *testing.T) {
 
 	require.True(t, diffrent)
 
+	ctx = testkeeper.AdvanceEpoch(ctx, keepers)
+
+	// test that in different epoch we get different pairings for consumer1
+	providers11, err := keepers.Pairing.GetPairingForClient(sdk.UnwrapSDKContext(ctx), spec.Index, consumer1.Addr)
+	require.Nil(t, err)
+
+	require.Equal(t, len(providers1), len(providers11))
+	for i := range providers1 {
+		require.NotEqual(t, providers1[i].Address, providers11[i].Address)
```

Comment:
this has a higher chance to fail than intended, require that only one of them at least is different not that all